commit e1b43bd556a611584a65f529e5077c1b54ace4f7
Author: Tony Luck <tony.luck@intel.com>
Date:   Mon Feb 5 15:47:43 2007 -0800

    [IA64] Fix example error injection program
    
    Progam accessed using /sys/devices/system/node/node0/cpu%d/err_inject/
    This path only exists for CONFIG_NUMA=y systems.  Better to use
    /sys/devices/system/cpu/cpu%d/err_inject/ which is available on all
    systems.
    
    Signed-off-by: Tony Luck <tony.luck@intel.com>

diff --git a/Documentation/ia64/err_inject.txt b/Documentation/ia64/err_inject.txt
index 26487c172cfc..6449a7090dbb 100644
--- a/Documentation/ia64/err_inject.txt
+++ b/Documentation/ia64/err_inject.txt
@@ -111,7 +111,7 @@ err_injection_tool.c:
 #define	ERR_DATA_BUFFER_SIZE 	3	// Three 8-byte.
 #define PARA_FIELD_NUM		5
 #define MASK_SIZE		(NR_CPUS/64)
-#define PATH_FORMAT "/sys/devices/system/node/node0/cpu%d/err_inject/"
+#define PATH_FORMAT "/sys/devices/system/cpu/cpu%d/err_inject/"
 
 int sched_setaffinity(pid_t pid, unsigned int len, unsigned long *mask);